packages/strapi-parts/src/Popover/Popover.stories.mdx

```diff
@@ -149,3 +159,10 @@ Description...
     }}
   </Story>
 </Canvas>
+
+
+## Props
+
+<!-- TODO: how do we show the props for PopoverContent?-->
+
```

Comment:
in this specific case it would be more interesting to display popoverContent's props (since popover doesnt have any) but im not sure if that's possible

Reply:
Maybe you can try to customise the ArgsTable?

Reply:
i'll do that while working on the Link/ LinkButton fixes